packages/jaeger-ui/src/components/DependencyGraph/DAG.tsx

```diff
@@ -34,16 +37,33 @@ type TProps = {
   selectedService: string;
   uiFind?: string;
   onMatchCountChange?: (count: number) => void;
+  onServiceSelect?: (service: string) => void;
 };
 
-export const renderNode = (vertex: TVertex, selectedService: string | null, uiFind?: string): ReactNode => {
+export const renderNode = (
```

Comment:
is it good practice for a functional component to take positional arguments instead of an object form?